envoy/stream_info/stream_info.h

```diff
@@ -91,8 +91,10 @@ enum ResponseFlag {
   OverloadManager = 0x2000000,
   // DNS resolution failed.
   DnsResolutionFailed = 0x4000000,
+  // Drop certain percentage of overloaded traffic.
+  DropOverLoad = 0x8000000,
```

Comment:
should have a docs change in https://www.envoyproxy.io/docs/envoy/latest/configuration/observability/access_log/usage with the others